rsyslog/Sanity/gnutls-openssl/main.fmf

```diff
@@ -2,55 +2,239 @@ summary: a sanity test for TLS communication
 contact: Dalibor Pospíšil <dapospis@redhat.com>
 test: ./runtest.sh
 require+:
-- library(distribution/dpcommon)
-- library(ControlFlow/ConditionalPhases)
-- /usr/bin/tshark
-- /usr/bin/certtool
+  - library(distribution/dpcommon)
+  - library(ControlFlow/ConditionalPhases)
+  - /usr/bin/tshark
+  - /usr/bin/certtool
 duration: 15m
 enabled: true
 tag:
-- Tier1
+  - Tier1
 tier: '1'
 adjust:
--   enabled: false
+  - enabled: false
     when: distro < rhel-8.5
+#/all:
+#    require+:
+#      - rsyslog-gnutls
+#      - rsyslog-openssl
+#    /old-syntax:
+#        environment+:
+#            rsyslogSyntax: old
+#        /x509_name:
+#            extra-summary: rsyslogSyntax=old /rsyslog/Sanity/gnutls-openssl
+#        /client_anon:
+#            environment+:
+#              CLIENT_NO_CERT: 1
+#            extra-summary: rsyslogSyntax=old CLIENT_NO_CERT=1 /rsyslog/Sanity/gnutls-openssl
+#        /server_anon:
+#            environment+:
+#              SERVER_NO_CERT: 1
+#            extra-summary: rsyslogSyntax=old SERVER_NO_CERT=1 /rsyslog/Sanity/gnutls-openssl
+#        /both_anon:
+#            environment+:
+#              CLIENT_NO_CERT: 1
+#              SERVER_NO_CERT: 1
+#            extra-summary: rsyslogSyntax=old SERVER_NO_CERT=1 CLIENT_NO_CERT=1 /rsyslog/Sanity/gnutls-openssl
+#    /new-syntax:
+#        environment+:
+#            rsyslogSyntax: new
+#        /x509_name:
+#            extra-summary: rsyslogSyntax=new /rsyslog/Sanity/gnutls-openssl
+#        /client_anon:
+#            environment+:
+#              CLIENT_NO_CERT: 1
+#            extra-summary: rsyslogSyntax=new CLIENT_NO_CERT=1 /rsyslog/Sanity/gnutls-openssl
+#        /server_anon:
+#            environment+:
+#              SERVER_NO_CERT: 1
+#            extra-summary: rsyslogSyntax=new SERVER_NO_CERT=1 /rsyslog/Sanity/gnutls-openssl
+#        /both_anon:
+#            environment+:
+#              CLIENT_NO_CERT: 1
+#              SERVER_NO_CERT: 1
+#            extra-summary: rsyslogSyntax=new SERVER_NO_CERT=1 CLIENT_NO_CERT=1 /rsyslog/Sanity/gnutls-openssl
```

Comment:
Is all this code pretended to be left commented for some reason?

Reply:
yes it was, I used that just for testing purposes, it is actually all covered by the individual virtual test cases